renovate.json

```diff
@@ -0,0 +1,72 @@
+{
+	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
+	"extends": ["config:best-practices"],
```

Comment:
@dsotirakis Should we explicitly mention any Grafana presets, or is the current form sufficient?

Reply:
@codebien 👋 
Grafana presets will be enabled by default (they are in the global config that our self-hosted solution uses).
You can explicitly disable them though if something doesn't work for you. 
https://github.com/grafana/grafana-renovate-config, which will soon be public 😄

Reply:
```suggestion
```
Hey @dsotirakis, thanks for the support. :bow: 

If I understand correctly, there isn't any need for defining this additional entry to the config. I'm dropping it.